app/routes-config.js

```diff
@@ -58,9 +58,13 @@ export const questionView = (id, answerId) =>
 export const expertPostView = (id, answerId) =>
   answerId ? `/experts/${id}#${uniqueAnswerId(answerId)}` : `/experts/${id}`;
 
-export const tutorialView = id => `/tutorials/${id}`;
+export const tutorialView = (id, answerId) =>
+  answerId
+    ? `/tutorials/${id}#${uniqueAnswerId(answerId)}`
```

Comment:
`/tutorials/` can be moved to local constant and replaced here and in the second optional chaining param too via string template